build.gradle.kts

```diff
@@ -37,7 +38,19 @@ buildscript {
   }
 }
 
-apply(from = rootProject.file(".buildscript/binary-validation.gradle"))
```

Comment:
Inlined this, since it's only used once and factoring it out just made it harder to read.

Reply:
👍 I'd copy pasted this from workflow not knowing how to do what you just did